app/models/refinery/jobs/job.rb

```diff
@@ -58,6 +60,13 @@ def find_by_slug_or_id(slug_or_id)
             find(slug_or_id)
           end
         end
+
+        def published_before(date=DateTime.now)
```

Comment:
Surrounding space missing in default value assignment.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.